core/commands/add.go

```diff
@@ -3,11 +3,13 @@ package commands
 import (
 	"fmt"
 	"io"
+	"os"
 	"path"
 	"strings"
 
 	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/cheggaaa/pb"
 	context "github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+	ignore "github.com/sabhiram/go-git-ignore"
```

Comment:
Obviously doesn't work from Windows, will setting this up on my Linux VM and going from there.